src/main/java/ro/msg/learning/shop/services/findingLocatonsStrategy/FindingLocationsStrategyType.java

```diff
@@ -1,6 +0,0 @@
-package ro.msg.learning.shop.services.findingLocatonsStrategy;
-
-public enum FindingLocationsStrategyType {
-    SingleLocation,
```

Comment:
why delete :( ? just make them upper case. Constants are a nice to have thing. It reduces the amount of plain written strings that pop out of nowhere. If you need to use the lowercase version, the enum has the capacity of holding a value that can look however you want. 
In the future they might even get a javdoc explaining what they do.